### ISSUE TITLE
Should defaultIfEmpty be also excluded in rxjs-no-unsafe-takeuntil by default?

### ISSUE THREAD

**Issue by zaripych** [cartant/rxjs-tslint-rules#71]
I've got a chunk of code which looks like below:

```
const latestInputAfterATrimOrInputCompletion = forkJoin(
      input,
      trims.pipe(
        take(1),
        takeUntil(input.pipe(
            ignoreElements(),
            endWith('completed')
          )
        ),
        defaultIfEmpty('STOPPED'),
      )
    );
```

I'm getting an error from `rxjs-no-unsafe-takeuntil` rule for `defaultIfEmpty` operator. When put above `takeUntil` in the chain the observable never completes. I believe the `defaultIfEmpty` is meant to be last in the chain - it seems similar to `isEmpty`, `toArray` and other operators already in the list of exceptions here https://github.com/cartant/rxjs-tslint-rules/blob/master/source/rules/rxjsNoUnsafeTakeuntilRule.ts.

I've put together a codepen where I could reproduce this. https://codepen.io/zaripych/pen/vvamoo?editors=1011

**Comment by cartant** [comment]
Yeah, `defaultIfEmpty` and [`concatIfEmpty`](https://github.com/ReactiveX/rxjs/issues/3714) should be added to [the list](https://github.com/cartant/rxjs-tslint-rules/blob/v4.14.3/source/rules/rxjsNoUnsafeTakeuntilRule.ts#L41-L62).

**Comment by cartant** [comment]
In the interim, you can add your own list to the rule's configuration in your `tslint.json`. See https://github.com/cartant/rxjs-tslint-rules/blob/master/test/v6/fixtures/no-unsafe-takeuntil/allow-custom/tslint.json

**Comment by cartant** [comment]
This should be fixed in 4.14.4.
